packages/tab/TabRipple.tsx

```diff
@@ -30,6 +30,7 @@ import {
 
 export interface TabRippleProps extends React.HTMLAttributes<HTMLDivElement>, Ripple.InjectedProps<HTMLDivElement> {
   className: string;
+  ref?: React.RefObject<any>,
```

Comment:
this seems unnecessary

Reply:
I moved it to the withRipple interface, but it is still needed because some components (Tab, for example, provides `ref` attribute)